### ISSUE TITLE
Downloading history with no progress

### ISSUE THREAD

**Issue by hnizamoglu** [ledgerwatch/erigon#8383]
Hi all,
Application has completed all the stages, but now it's not increasing the latest block. When I send `get_block` rpc value it doesn't change and shows `18273232`. Below you can see the portion of the logs. Besides these logs, there is not much going on. I tried restarting the node several times but it always stuck with these logs. Any idea about it?
```
INFO[10-05|12:56:46.800] P2P                                      app=caplin peers=30
INFO[10-05|12:56:47.000] Downloading History                      app=caplin stage=DownloadHistoricalBlocks progress=7471328 blk/sec=0.0 peers=30
INFO[10-05|12:57:12.681] [p2p] GoodPeers                          eth68=14 eth67=23
INFO[10-05|12:57:13.607] [txpool] stat                            pending=9402 baseFee=24539 queued=30000 alloc=4.7GB sys=8.7GB
INFO[10-05|12:57:17.000] Downloading History                      app=caplin stage=DownloadHistoricalBlocks progress=7471328 blk/sec=0.0 peers=32
INFO[10-05|12:57:46.800] P2P                                      app=caplin peers=32
INFO[10-05|12:57:46.999] Downloading History                      app=caplin stage=DownloadHistoricalBlocks progress=7471328 blk/sec=0.0 peers=32
INFO[10-05|12:58:13.597] [txpool] stat                            pending=9469 baseFee=24601 queued=30000 alloc=4.2GB sys=8.7GB
INFO[10-05|12:58:17.000] Downloading History                      app=caplin stage=DownloadHistoricalBlocks progress=7471328 blk/sec=0.0 peers=32
INFO[10-05|12:58:46.800] P2P                                      app=caplin peers=27
INFO[10-05|12:58:46.999] Downloading History                      app=caplin stage=DownloadHistoricalBlocks progress=7471328 blk/sec=0.0 peers=27
INFO[10-05|12:59:12.681] [p2p] GoodPeers                          eth68=15 eth67=23
INFO[10-05|12:59:13.606] [txpool] stat                            pending=9499 baseFee=24698 queued=30000 alloc=4.6GB sys=8.7GB
INFO[10-05|12:59:17.000] Downloading History                      app=caplin stage=DownloadHistoricalBlocks progress=7471328 blk/sec=0.0 peers=32
INFO[10-05|12:59:46.799] P2P                                      app=caplin peers=30
INFO[10-05|12:59:47.000] Downloading History                      app=caplin stage=DownloadHistoricalBlocks progress=7471328 blk/sec=0.0 peers=30
INFO[10-05|13:00:13.597] [txpool] stat                            pending=9682 baseFee=24806 queued=30000 alloc=4.3GB sys=8.7GB
INFO[10-05|13:00:17.000] Downloading History                      app=caplin stage=DownloadHistoricalBlocks progress=7471328 blk/sec=0.0 peers=26
INFO[10-05|13:00:46.799] P2P                                      app=caplin peers=30
INFO[10-05|13:00:47.000] Downloading History                      app=caplin stage=DownloadHistoricalBlocks progress=7471328 blk/sec=0.0 peers=30
INFO[10-05|13:01:12.681] [p2p] GoodPeers                          eth68=15 eth67=23
INFO[10-05|13:01:13.605] [txpool] stat                            pending=9730 baseFee=24913 queued=30000 alloc=4.7GB sys=8.7GB
INFO[10-05|13:01:17.000] Downloading History                      app=caplin stage=DownloadHistoricalBlocks progress=7471328 blk/sec=0.0 peers=28
INFO[10-05|13:01:46.799] P2P                                      app=caplin peers=26
INFO[10-05|13:01:47.000] Downloading History                      app=caplin stage=DownloadHistoricalBlocks progress=7471328 blk/sec=0.0 peers=26
INFO[10-05|13:02:13.598] [txpool] stat                            pending=9799 baseFee=24966 queued=30000 alloc=4.3GB sys=8.7GB
INFO[10-05|13:02:17.000] Downloading History                      app=caplin stage=DownloadHistoricalBlocks progress=7471328 blk/sec=0.0 peers=28

```

**Comment by wmitsuda** [comment]
I do confirm seeing something similar on 2.49.3/sepolia, but was able to workaround it by restarting.

**Comment by 0xCryptris** [comment]
Same issue here with mainnet in Docker. Restart did not solve the issue.

**Comment by alexpeterson91** [comment]
@hnizamoglu did you solve this issue? if so please share with others still experiencing it, such as myself. Thanks!

**Comment by 0xCryptris** [comment]
As I didn't receive any feedback and found no solution, I went back to docker image: thorax/erigon:v2.48.1 and bite the bullet of a full resync for several days. Works w/o any issues since then.

**Comment by alexpeterson91** [comment]
Got it thanks. Doesn't do me any good since I need newer releases that cover several needed updates for AuRa chains like gnosis chain which is what I'm working on atm. But I'm doing a full resync now. I forgot I had finished syncing on a different image than I started with due to changing settings in a custom docker container based on the thorax image.  I'm not sure If the beacon snapshot or torrent for gnosis beacon is seeded properly based on some discussion I saw in the Erigon discord. So I may be trying in vain but hey it's not that hard just lots of waiting. lol.
